web/client/utils/LayersUtils.js

```diff
@@ -717,6 +717,7 @@ export const saveLayer = (layer) => {
     layer.localizedLayerStyles ? { localizedLayerStyles: layer.localizedLayerStyles } : {},
     layer.options ? { options: layer.options } : {},
     layer.credits ? { credits: layer.credits } : {},
+    layer.security ? { security: layer.security } : {},
```

Comment:
Just for confirmation, does this property contain only the sourceId and type?

Reply:
afaik yes